docs/gemini/gemini.rst

```diff
@@ -83,6 +83,20 @@ and the program ID.  For a complete list of available search fields, see
                           0.0           Full Frame                         -- Gemini-North    True ...
                           0.0           Full Frame                         -- Gemini-North    True ...
 
+In addition, the criteria query can accept additional parameters via the `args` and `kwargs` optional parameters.
```

Comment:
double backticks

Reply:
also, might be nice to have another extra example with passing in the kwargs

Reply:
done

Reply:
> also, might be nice to have another extra example with passing in the kwargs

I missed this but made a note of it for next time